docs/multicluster/user-guide.md

```diff
@@ -396,18 +396,12 @@ $kubectl annotate node node-1 multicluster.antrea.io/gateway=true
 ```
 
 Multi-cluster Controller in the member cluster will detect the Gateway Node, and
-create a `Gateway` CR with the same name as the Node. You can check the `Gateway`
-with: `kubectl get gateway node-1 -o yaml`, which should show the Gateway
-information like:
+create a `Gateway` CR with the same name as the Node. You can check it with command:
 
-```yaml
-apiVersion: multicluster.crd.antrea.io/v1alpha1
-kind: Gateway
-metadata:
-  name: node-1
-  namespace: kube-system
-gatewayIP: 10.17.27.55
-internalIP: 10.17.27.55
+```bash
+$kubectl get gateway -n kube-system
+NAME          GATEWAY IP     INTERNAL IP    AGE
+node-1        10.17.27.55    10.17.27.55    1m
```

Comment:
"1m" -> "10s" to be consistent with other commands?

Reply:
Done